eng/build.yml

```diff
@@ -1,65 +1,143 @@
 parameters:
   # Agent OS identifier and used as job name
-  agentOs: ''
+- name: agentOs
+  type: string
 
-  # Agent pool
-  pool: {}
+  # Job name
+- name: jobName
+  type: string
 
-  # Additional variables
-  variables: {}
-
-  # Build strategy - matrix
-  strategy: {}
+# Container to run the build in, if any
+- name: container
+  type: string
+  default: ''
 
   # Job timeout
-  timeoutInMinutes: 180
-
-  # Publish using pipelines
-  enablePublishUsingPipelines: true
-
-phases:
+- name: timeoutInMinutes
+  type: number
+  default: 180
+
+# Build configuration (Debug, Release)
+- name: buildConfiguration
+  type: string
+  values:
+  - Debug
+  - Release
+
+# Build architecture
+- name: buildArchitecture
+  type: string
+  values:
+  - arm
+  - arm64
+  - x64
+  - x86
+  
+# Linux portable. If true, passes portable switch to build
+- name: linuxPortable
+  type: boolean
+  default: false
+
+# Runtime Identifier
+- name: runtimeIdentifier
+  type: string
+  default: ''
+
+# UI lang
+- name: dotnetCLIUILanguage
+  type: string
+  default: ''
+
+# Additional parameters
+- name: additionalBuildParameters
+  type: string
+  default: ''
+
+# Run tests
+- name: runTests
+  type: boolean
+  default: true
+
+# PGO instrumentation jobs
+- name: pgoInstrument
+  type: boolean
+  default: false
+
+jobs:
 - template: /eng/common/templates/job/job.yml
   parameters:
+    # Set up the name of the job.
     ${{ if parameters.pgoInstrument }}:
-      name: PGO_${{ parameters.agentOs }}
+      name: PGO_${{ parameters.agentOs }}_${{ parameters.jobName }}
     ${{ if not(parameters.pgoInstrument) }}:
-      name: ${{ parameters.agentOs }}
-    timeoutInMinutes: ${{ parameters.timeoutInMinutes }}
+      name: ${{ parameters.agentOs }}_${{ parameters.jobName }}
+    
+    ## Set up the pool/machine info to be used based on the Agent OS
     ${{ if eq(parameters.agentOs, 'Windows_NT') }}:
       enableMicrobuild: true
+      pool:
+        ${{ if eq(variables['System.TeamProject'], 'public') }}:
+          name: NetCore-Svc-Public
```

Comment:
Shouldn't this just be NetCore-Public since this is main?

Reply:
Same for all the pools. Svc is only for release branches

Reply:
Thanks..not sure how that got changed.